### ISSUE TITLE
Quantity update: Wrong config for icons.

### ISSUE THREAD

**Issue by kruchy8** [SnowdogApps/magento2-alpaca-components#293]
For both `minus` and `plus` , it should wrap `iconClass` and `iconId` into `icon` property inside of object. Instead of:

```
minusQtyButton: {
      tag: 'button',
      class: 'button--icon quantity-update__button quantity-update__button--minus quantity-update__button--disabled',
      attributes: 'type="button" aria-label="quantity minus button"',
      iconClass: 'button__icon quantity-update__icon',
      iconId: 'minus'
    },
```

it should be:

```
    minusQtyButton: {
      tag: 'button',
      class: 'button--icon quantity-update__button quantity-update__button--minus quantity-update__button--disabled',
      attributes: 'type="button" aria-label="quantity minus button"',
      icon: {
        iconClass: 'button__icon quantity-update__icon',
        iconId: 'minus'
      }
    },
```

**Comment by Januszpl** [comment]
if it's really necessary to change than imho we should also rename in second example `iconClass` and `iconId` to `class`, `icon` no to have later sth like `icon.iconClass`

**Comment by anthuanvasquez** [comment]
Fixed https://github.com/SnowdogApps/magento2-alpaca-components/pull/471

**Comment by szafran89** [comment]
FIxed and merged to develop #471
